sklearn/ensemble/_hist_gradient_boosting/binning.py

```diff
@@ -151,12 +152,14 @@ def __init__(
         is_categorical=None,
         known_categories=None,
         random_state=None,
+        n_threads=None,
```

Comment:
Even if this is a private class, we probably need to update the docstring to document this param. I think it's important to tell that this takes cgroups CPU quotas into account when `n_threads=None`.